core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel.hpp

```diff
@@ -172,22 +172,15 @@ struct ParallelReduceSpecialize<FunctorType, Kokkos::RangePolicy<PolicyArgs...>,
 
     ValueType result = ValueType();
 
-    // FIXME_OPENMPTARGET : For a single range parallel loop, the number of
-    // teams (league_size) should be chosen by the compiler for optimal
-    // performance based on the architecture.
-    // However currently we have hardcoded it to 512 as it has shown the best
-    // performance for some micro-benchmarks with LLVM compiler on NVIDIA V100.
```

Comment:
When was that fixed?

Reply:
Patrick informed me that this was fixed with the latest intel compilers.